File: CommonBase/src/main/java/com/common/base/callback/SimpleCallback0.java
package com.common.base.callback;

/**
 * 简化回调接口
 *
 * @author LiuFeng
 * @data 2020/7/16 11:05
 */
public class SimpleCallback0 implements CommonCallback0 {

    @Override
    public void onSuccess() {}

    @Override
    public void onError(int code, String desc) {}
}

File: CommonBase/src/main/java/com/common/base/callback/SimpleCallback0.kt
package com.common.base.callback

/**
 * 简化回调接口
 *
 * @author LiuFeng
 * @data 2020/7/16 11:05
 */
class SimpleCallback0 : CommonCallback0 {
    override fun onSuccess() {}
    override fun onError(code: Int, desc: String?) {}
}